src/main/java/lv/ctco/javaschool/app/control/exceptions/UserNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package lv.ctco.javaschool.app.control.exceptions;
+
+public class UserNotFoundException extends Exception {
```

Comment:
Please change to:
`public class UserNotFoundException extends RuntimeException {`

Reply:
inherited UserNotFoundException from RuntimeException and removed throwing UserNotFoundException from getTripPassengersByTripId() method in the TriApi class